steering_vectors/steering_vector.py

```diff
@@ -101,12 +105,13 @@ def patch_activations(
             module = get_module(model, layer_name)
             handle = module.register_forward_hook(
                 # create the hook via function call since python only creates new scopes on functions
-                _create_additive_hook(
+                _create_additive_hook_output(
                     target_activation.reshape(1, 1, -1), token_indices, operator
                 )
             )
+            module_names.append(layer_name + ".output")
```

Comment:
Where does this `.output` string come from? It seems like not all possible modules we could steer will have submodule called "output"?

Reply:
This is based on the observation that the `create_additive_hook` function modifies the output. I feel it's important to clarify that, as otherwise it's ambiguous whether we're steering at layer input vs layer output.

Reply:
But is there an actual module called "output" inside of every module we're applying steering? It looks like this string is also never being read out?

Reply:
There is not an actual module called "output". I agree it's misleading given the syntax because it implies the torch module has an 'output' module. open to suggestions on how to improve the naming convention. 

It's intended to be read by the user

Reply:
Is the issue that the built-in torch `RemovableHandle` class doesn't include the name of the module it's corresponding to? If we think that's a problem, I'd vote to just put the `layer_name` in the list instead of appending `.output`. It still feels weird to me to add this field we're never reading, but it would at least be corresponding to the actual module then.

Reply:
Yeah, but in that case I still think it's a problem that it's not uniquely specified whether we're referring to the output or the input.

Reply:
I'm still confused what the use-case is here for adding this feature. It sounds like this is a form of documentation, to let the user know that the steering handle is editing the output of modules? If so, could this just be added to the documentation and explain that the steering handle is editing the output of modules? I'm not a huge fan of adding unused code as a form of documentation in the hope that the user inspects the object, but if you feel this is necessary, could the field you choose express this somehow? Something like `edit_module_outputs` or something?